src/features/decorators/common.ts

```diff
@@ -0,0 +1,49 @@
+import { Replace, OptionsObject } from "../feature";
+import { Computable, GetComputableType, processComputable, ProcessedComputable } from "util/computed";
+import { Persistent, State } from "game/persistence";
+
+/*----====----*/
```

Comment:
I'm not a huge fan of adding these separators and #region comments since nowhere else in the project has them